### PR TITLE
Add `display.markdown` method

### PR DESCRIPTION
**Pull request by colinking**
Adds a new method for creating markdown displays:

```ts
await airplane.display.markdown(`Loading...`)
```

Resolves AIR-4286, AIR-4287

### CODE REVIEW

**Comment by colinking** [comment]
Current dependencies on/for this PR:
* main
  * **PR #79** <a href="https://app.graphite.dev/github/pr/airplanedev/node-sdk/79" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/node-sdk/79?utm_source=stack-comment).

File: internal/api/types.ts [hunk 1]
```diff
@@ -77,3 +77,19 @@ export type Prompt = {
...
 };
+
+type DisplayBase = {
+  id: string;
+  createdAt: string;
+};
+
+export type Display = DisplayBase & DisplayConfig;
+
+export type DisplayConfig = DisplayMarkdownConfig;
```
Comment (ameyakhare) [inline internal/api/types.ts:88]: why this renaming?

**Review by ameyakhare — APPROVED** [review]
lgtm

File: internal/api/types.ts [hunk 2]
```diff
@@ -77,3 +77,19 @@ export type Prompt = {
   values: ParamValues;
   submittedAt: string | null;
 };
+
+type DisplayBase = {
+  id: string;
+  createdAt: string;
```
[the remark this answers is not in the document]
Comment (colinking) [reply internal/api/types.ts:83]: `DisplayConfig` will always have a `kind`, so it doesn't need one.

File: internal/api/types.ts (hunk 1 above)
Comment (colinking) [reply internal/api/types.ts:88]: This will eventually be a union of other kinds of displays.

**Pull request merged**